policyengine_us/parameters/gov/states/mi/tax/income/deductions/retirement_benefits/expanded/rate.yaml

```diff
@@ -0,0 +1,17 @@
+description: Michigan calculates expanded deduction for retirement and pension benefits through this rate.  
```

Comment:
Could you say something like 
"Michigan multiplies x by this rate under the expanded retirement and pension benefits deduction.